next.clayui.com/content/docs/components/Panel.mdx

```diff
@@ -21,6 +21,11 @@ import {Panel} from '../../../src/components/clay/Panel';
 </div>
 </div>
 
+<div class="clay-site-alert alert alert-warning">
+	<strong class="lead">Warning</strong>
+	Before you get started with this doc, read about <a href="/foundations/composing.html" target="_blank">Clay's fundamentals of composition</a> to move on. The information below is assuming you have read about <a href="/foundations/composing.html" target="_blank">Clay's compositional philosophy</a> and learned about the terms.
```

Comment:
```suggestion
	Before you get started with this doc, read about <a href="/docs/foundations/composing.html" target="_blank">Clay's fundamentals of composition</a> to move on. The information below is assuming you have read about <a href="/docs/foundations/composing.html" target="_blank">Clay's compositional philosophy</a> and learned about the terms.
```

I've been talking to @bryceosterhaus about this, so let's evaluate if it's really necessary to add this to all documents.